tests/utils_test.py

```diff
@@ -0,0 +1,22 @@
+"""Unit tests for english language checker"""
+import unittest
+import libcipher.utils as utils
+
+filepath = '/home/travis/build/Chennaipy/libcipher/libcipher/dictionary.txt'
```

Comment:
@bravegnu , @shrayasr: I still am unable to follow. If I need to put the contents of dictionary file into the ```english_words``` list, it would have 45333 words. Should I do that, and then use the StringIO class to create the object dict_file and pass this to the 
```Englishchecker``` constructor?
Thank you